src/main/kotlin/com/terraformation/backend/accelerator/api/ParticipantProjectSpeciesController.kt

```diff
@@ -179,6 +180,7 @@ data class ParticipantProjectForSpeciesPayload(
     val activeDeliverableId: DeliverableId?,
     val participantProjectSpeciesId: ParticipantProjectSpeciesId,
     val participantProjectSpeciesSubmissionStatus: SubmissionStatus,
+    val participantProjectSpeciesNativeCategory: SpeciesNativeCategory?,
```

Comment:
You can ditch this and line 194, in [this PR](https://github.com/terraware/terraware-server/pull/1993) I return back the entire `participantProjectSpecies` model in this response payload.

Reply:
Just saw you did that! Nice!

Reply:
@nickgraz I think I still need this change, because I see now that the `participantProjectSpecies` is added in `SpeciesForParticipantProjectPayload` but I need it in `ParticipantProjectForSpeciesPayload` too